extensions-core/multi-stage-query/src/main/java/org/apache/druid/msq/querykit/WindowOperatorQueryFrameProcessor.java

```diff
@@ -187,14 +184,12 @@ public ReturnOrAwait<Object> runIncrementally(IntSet readableInputs)
      *     We might think to reimplement them in the MSQ way so that we do not have to materialize so much data
      */
 
-    // Phase 1 of the execution
-    // eagerly validate presence of empty OVER() clause
-    if (isOverEmpty) {
```

Comment:
why there are no tests which have broken?
I don't see any new tests which put expectations on the plans....

wouldn't all the added q/e tests pass without these changes?

Reply:
> why there are no tests which have broken?

Why would you expect any tests to break? These changes aren't expected to change query results.

> I don't see any new tests which put expectations on the plans....

Right, I was hoping to finalise the query kit changes before I add tests for asserting the generated MSQ spec. I was thinking of doing that in a separate future PR, if that works?

> wouldn't all the added q/e tests pass without these changes?

Yes, but that's expected, right?

Reply:
@kgyrtkirk As per offline discussion, have reverted changes, back to creating a single stage for queries having empty over clause.

Appreciate if you could check the PR again, thanks!

Reply:
I did know that some coverage is missing in this area - but I wasn't expecting that there will be no such covereage :)